gsitemap.php

```diff
@@ -106,6 +107,7 @@ public function install()
             'GSITEMAP_PRIORITY_HOME' => 1.0,
             'GSITEMAP_PRIORITY_PRODUCT' => 0.9,
             'GSITEMAP_PRIORITY_CATEGORY' => 0.8,
+			'GSITEMAP_PRIORITY_MANUFACTURER' => 0.7,
```

Comment:
Bad indentation

Reply:
Sorry for identation errors, didn't notice. Will check it before next time. Thanks!